commands/translate.py

```diff
@@ -0,0 +1,169 @@
+from commands.base import Command
+from discord import Embed
+from helpers import *
+from utils.embed_table import EmbedTable
+
+import googletrans
+import json
+import re
+
+HISTORY = 10
+HISTORY_LIMIT = 100
+FLAGS_FILE = "files/flags.json"
+AVATAR_FORMAT = "https://cdn.discordapp.com/avatars/{0.id}/{0.avatar}.png?size=128"
+TRANSLATE_COLOR = int('4885ed', 16)
+
+class Translate(Command):
+    desc = """Translates a marked message to another language (removes all emojis).
+    React with a flag and type `!translate` to translate the message to the corresponding language.
+    Optional `index` argument defines offset of message to translate in list of marked messages."""
+
+    async def eval(self, index=0):
+        if int(index) < 0:
```

Comment:
Breaks if `index` is not a number (you'll need to catch the ValueError, take a look at archive.py)